src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -221,12 +230,13 @@ class DropdownFilterAjax extends DropdownFilter {
       .query(this.props.additionalRequestParams)
       .end(this.ajaxUpdateList);
   }
-
   /**
    * Applies some data from AJAX to the list
    */
   ajaxUpdateList = (err, response) => {
-    this.updateList(response.body.data[0]);
+    this.updateList(
+      this.props.formatResponse ? this.props.formatResponse(response.body.data[0]) : response.body.data[0]
```

Comment:
Should we just pass response or response.body here as there may be custom logic depending on if a error occurred and we don't know for sure that data will be an array?